tests/test_the_test/test_version.py

```diff
@@ -85,7 +82,7 @@ def test_version_serialization():
     assert v == "7.43.1"
 
     v = Version("7.50.0-dbm-oracle-0.1", "agent")
-    assert str(v) == "7.50.0+dbm.oracle.0.1"
+    assert str(v) == "7.50.0-dbm-oracle-0.1"
 
```

Comment:
```suggestion

v = Version("1.2.3-pre", "nodejs")
assert str(v) == "1.2.3-pre"

```

Reply:
There is a test in line 78:
```python
    v = Version("3.0.0pre0", "nodejs")
    assert v == "3.0.0pre0"
```
but this is not a valid semver version, so it will never happen for Node, this test is bogus i think